test/components/KonnectorTile.spec.jsx

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+/* eslint-env jest */
+
+import React from 'react'
+import { configure, shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-15'
+
+import KonnectorTile from '../../src/components/KonnectorTile'
```

Comment:
Test will be better directly on `KonnectorTile` component. Here we are importing the redux-connected component.

So the import instruction should be
```
import { KonnectorTile } from '../../src/components/KonnectorTile'
```